fossunited/volunteering/doctype/event_volunteer/event_volunteer.json

```diff
@@ -58,18 +61,56 @@
    "fieldname": "city",
    "fieldtype": "Link",
    "label": "City",
-   "options": "City",
-   "reqd": 1
+   "options": "City"
   },
   {
    "fieldname": "identification",
    "fieldtype": "Data",
    "label": "I am"
+  },
+  {
+   "fieldname": "status_info_section",
+   "fieldtype": "Section Break",
+   "label": "Status Info"
+  },
+  {
+   "fieldname": "status",
+   "fieldtype": "Select",
+   "label": "Status",
+   "options": "\nSelected\nWaitlisted\nRejected"
+  },
```

Comment:
status of what?

Reply:
Whether they are selected to volunteer - only volunteers with selected status would be enlisted in website.

Reply:
Accepted woold be better than selected

Reply:
I agree - changed it